packages/lesswrong/components/recentDiscussion/RecentDiscussionSubscribeReminder.tsx

```diff
@@ -71,33 +82,40 @@ const RecentDiscussionSubscribeReminder = ({classes}: {
   const [hide, setHide] = useState(false);
   const [subscribeChecked, setSubscribeChecked] = useState(true);
   const [subscriptionConfirmed, setSubscriptionConfirmed] = useState(false);
-  const [emailAddressInput, setEmailAddressInput] = useState("");
+  const emailAddressInput = useRef<HTMLInputElement|null>(null);
   const [loading, setLoading] = useState(false);
   const { flash } = useMessages();
   const {WrappedLoginForm, SignupSubscribeToCurated, Loading, AnalyticsInViewTracker } = Components;
-  const subscriptionDescription = '(2-3 posts per week, selected by the LessWrong moderation team.)'
   const { captureEvent } = useTracking({eventProps: {pageElementContext: "subscribeReminder"}});
-  
+
   // Show admins a random version of the widget. Makes sure we notice if it's intrusive/bad.
   const [adminBranch, setAdminBranch] = useState(-1);
   const adminUiMessage = currentUser?.isAdmin ? <div className={classes.adminNotice}>
     You are seeing this UI element because you're an admin. Admins are shown a random version of the
     subscribe-reminder even if they're already subscribed, to make sure it still works and isn't
     annoying.
   </div>: null
-  
+
   useEffect(() => {
-    if (adminBranch == -1 && currentUser?.isAdmin) {
+    if (adminBranch === -1 && currentUser?.isAdmin) {
```

Comment:
Out of curiosity, what was happening here? I can't think of anything which is `== -1` but not `=== -1`, and I'm excited to learn about yet another insane JavaScript corner case.

Reply:
This was more of an instinct, since in my previous job we had a linting rule that forbade `==` and `!=`

Reply:
I want that linting rule